### ISSUE TITLE
Create Draft Release GitHub action

### ISSUE THREAD

**Issue by msashkin** [odessajavaclub/coffee-pot#51]
GitHub Actions links:
- Create draft release: https://github.com/release-drafter/release-drafter
- Download artifact: https://github.com/actions/download-artifact
- Create release: https://github.com/actions/create-release
- Upload release asset: https://github.com/actions/upload-release-asset
- Release notes: https://github.com/Decathlon/release-notes-generator-action

GitHub API docs:
- https://developer.github.com/v3/repos/releases/#create-a-release

It would be nice to have a separate GitHub action to be launched on tags (v*), which would create a release with corresponding artifacts.

An alternative approach is to create a draft release gh action which would create draft releases when PRs are merged to master. https://github.com/release-drafter/release-drafter implements that strategy.

GitHub Draft releases **won’t** be seen by the public unless they are published.

https://github.com/release-drafter/release-drafter requires a config file to be submitted first to master in order to add gh action. Thus, at least two PRs are required.

Labels for each PR should be set properly in order to make correct categories in releases

Drawbacks:
- In order to test Draft Release configuration, the configuration must be merged into master
https://github.com/release-drafter/release-drafter/issues/203
- It doesn't scan previous PRs. In other words, only after merging Draft Release configuration it begins to aggregate new PRs. The first 0.0.1 release should be created **manually**.
